package.json

```diff
@@ -58,6 +58,10 @@
     "wkx": "^0.5.0"
   },
   "devDependencies": {
+    "@types/inflection": "^1.13.0",
+    "@types/lodash": "^4.14.178",
+    "@types/uuid": "^8.3.3",
+    "js-combinatorics": "^0.6.1",
```

Comment:
The return of `js-combinatorics` is my bad. Merge error.
Pointing it out so we don't forget to remove it.

Reply:
And pin the types so we don't get a renovate PR for it